.azure-pipelines/windows-net461.yml

```diff
@@ -0,0 +1,47 @@
+parameters:
+  configuration: Debug
+
+steps:
+
+- task: NuGetCommand@2
+  displayName: nuget install xunit.runner.console
+  inputs:
+    command: custom
+    arguments: install xunit.runner.console
+
+- task: PowerShell@2
+  displayName: Set TargetFramework to net461 by force
+  inputs:
+    targetType: inline
+    script: |
+      foreach ($path in gci *.Tests/*.Tests.csproj) {
+        [xml]$csproj = Get-Content $path
+        $csproj.Project.PropertyGroup[0].TargetFramework = "net461"
+        $csproj.Save($path)
+      }
+
+- task: MSBuild@1
+  inputs:
+    solution: Libplanet.sln
+    msbuildVersion: "16.0"
+    configuration: ${{ parameters.configuration }}
+    msbuildArguments: /restore
+
+- task: Bash@3
+  displayName: xunit.console.exe *.Tests.dll
+  inputs:
+    targetType: inline
+    script: |
+      ${{ parameters.testPrefix }} \
+        xunit.runner.console.*/tools/net472/xunit.console.exe \
+          *.Tests/bin/$(configuration)/net*/*.Tests.dll \
+          ${{ parameters.testArguments }}
+    # FIXME: For unknown reason, on Windows tests depending on TURN_SERVER_URL
+    #        seems to never end or to take too long time.  We should diagnose
+    #        this and make Windows builds to run these tests too.
+    # FIXME: For unknown reason, on Windows + Mono SwarmTests seem never end.
```

Comment:
It may be [`AsyncIO.ForceDotnet.Force()`](https://github.com/somdoron/AsyncIO/blob/986d7424dff95aa0bc87d10cb56b85451c2d8981/Source/AsyncIO/ForceDotNet.cs#L11) related issue. I'll fix in next PR.